### PR TITLE
Remove StringIO wrapper

### PR DESCRIPTION
**Pull request by nijel**
It is confusing to have StringIO which is actually BytesIO.

### CODE REVIEW

File: translate/misc/wStringIO.py [hunk 1]
```diff
@@ -17,31 +17,17 @@
...
-class CatchStringOutput(StringIO):
+class CatchStringOutput(BytesIO):
     """catches the output before it is closed and sends it to an onclose method
     """
 
     def __init__(self, onclose):
         """Set up the output stream, and remember a method to call on closing
         """
-        StringIO.__init__(self)
+        super().__init__()
```
Comment (unho) [inline translate/misc/wStringIO.py:30]: I guess this is a `super` you missed in the other PR.

**Comment by unho** [comment]
@nijel lgtm

**Comment by nijel** [comment]
Thanks for review.

File: translate/misc/wStringIO.py (hunk 1 above)
Comment (nijel) [reply translate/misc/wStringIO.py:30]: Probably, it's easy to miss something in those significant changes.

**Pull request merged**